README.md

```diff
@@ -1,66 +1,14 @@
 ![commercetools logo](https://cdn.rawgit.com/commercetools/press-kit/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png)
-
+⛔ [Deprecated]
 # Stock export
```

Comment:
Perhaps you can also indicate that the name is changed in the new repo to Inventory export:

# Stock export (Inventory export)